architecture/additional_concepts/networking.adoc

```diff
@@ -92,9 +92,18 @@ routing for its endpoints. Service DNS can still be used and responds with
 multiple A records, one for each pod of the service, allowing the client to
 round-robin between each pod.
 
+[[network-plugins]]
+
+== Network Plugins
+
+{product-title} supports the same plugin model as Kubernetes for networking pods. 
+ifdef::openshift-enterprise,openshift-origin[]
+The following network plugins are currently supported by {product-title}.
```

Comment:
IMO this means that the playbooks need to be packaged and provided by nuage. This will help to make it clear that they support the install.

We should also make our install docs a reference to nuage docs that cover this information.